### PR TITLE
Small bugs & improvements

### PR DESCRIPTION
**Pull request by recone83**
Monster name length, MuMain was set to 32, but Land_End had names longer than 32 - caused crashes
Enabled Karutan

### CODE REVIEW

File: Source Main 5.2/source/MapManager.cpp [hunk 1]
```diff
@@ -1199,10 +1204,10 @@ void CMapManager::Load() // OK
 #ifdef ASG_ADD_MAP_KARUTAN
         else if (IsKarutanMap())
         {
-            Models[66].Actions[0].PlaySpeed = 0.15f;
-            Models[66].Actions[1].PlaySpeed = 0.15f;
+            //Models[66].Actions[0].PlaySpeed = 0.15f;
```
Comment (sven-n) [inline Source Main 5.2/source/MapManager.cpp:1207]: why these changes?

**Comment by sven-n** [comment]
I merged my other PR #25 which also enabled karutan 😅 I'll change monster name to length of 40, too.

File: Source Main 5.2/source/MapManager.cpp (hunk 1 above)
Comment (recone83) [reply Source Main 5.2/source/MapManager.cpp:1207]: Models does not exists, causes crash

**Comment by recone83** [comment]
> I merged my other PR #25 which also enabled karutan 😅 I'll change monster name to length of 40, too.

sweet, thanks
